tests/test_views.py

```diff
@@ -57,3 +71,17 @@ def test_static_recommendation(dummy_cache, client, static_recommendation_manage
     assert response.status_code == 200
     assert response['Content-Type'] == 'application/json'
     assert response.content == b'{"results": ["addon-1", "addon-2", "addon-N"]}'
+
+
+def test_locale_recommendation(dummy_cache, client, locale_recommendation_manager):
+    kwargs = {'client_id': str(uuid.uuid4())}
+    response = client.get(reverse('recommendations', kwargs=kwargs), {'locale': 'en-US'})
```

Comment:
Let's add another case within this function to test passing the 'platform' as well

Reply:
Thanks!  I added the platform test and updated the commit message now.